packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md

```diff
@@ -87,30 +87,36 @@ The following table describes which capabilities are supported by the Custom Aut
 | Machine-to-Machine or Microservices        | Yes                                  | No                          |
 | Mint Access Tokens with Okta API Scopes    | No                                   | Yes                         |
 
-## Key rotation for Custom Authorization Servers
+## Key rotation
 
-* For security purposes, Okta automatically rotates keys used to sign the token.
+Key rotation is when an encryption key is retired and replaced by generating a new cryptographic key. Rotating keys on a regular basis helps meet industry standards and cryptographic best practices.
 
-* The current key rotation schedule is four times a year. This schedule can change without notice.
+> **Note:** The current Okta key rotation schedule is four times a year, but can change without notice.
+
+If you are using a Custom Authorization Server, configure and perform key rollover/rotation at the [Authorization Server level](/docs/reference/api/authorization-servers/#credentials-object).
+
+If you are using the Org Authorization Server, configure and perform key rollover/rotation at the [client level](/docs/reference/api/apps/#generate-new-application-key-credential).
+
+### Key rotation for Custom Authorization Servers
+
+* For security purposes, Okta automatically rotates keys used to sign tokens.
 
 * In case of an emergency, Okta can rotate keys as needed.
 
 * Okta always publishes keys to the `jwks_uri`.
 
 * To save the network round trip, your app should cache the `jwks_uri` response locally. The [standard HTTP caching headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) are used and should be respected.
 
-* The administrator can switch the Authorization Server key rotation mode by updating the Authorization Server's `rotationMode` property. For more information see the API Reference: [Authorization Server Credentials Signing Object](/docs/reference/api/authorization-servers/#credentials-object).
+* You can switch the Authorization Server key rotation mode by updating the Authorization Server's `rotationMode` property. For more information see the API Reference: [Authorization Server Credentials Signing Object](/docs/reference/api/authorization-servers/#credentials-object).
 
 > **Caution:** Keys used to sign tokens automatically rotate and should always be resolved dynamically against the published JWKS. Your app might fail if you hardcode public keys in your applications. Be sure to include key rollover in your implementation.
 
-> **Note:** If your application can't retrieve keys dynamically, the administrator can disable the automatic key rotation in the administrator UI, [generate a key credential](/docs/reference/api/apps/#generate-new-application-key-credential), and [update the application](/docs/reference/api/apps/#update-key-credential-for-application) to use it for signing.
-
-## Key rotation for Org Authorization Servers
+### Key rotation for Org Authorization Servers
```

Comment:
nit: there is only a single Org Auth Server per org, so it might make sense for this section to be titled `Key rotation for the Org Authorization Server`